scripts/create-many-trial-session-cases.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+trialSessionId="86a18457-a7ba-45a8-a1e2-de9d6dcf2b0f"
+token="eyJraWQiOiJRQzAwUFRsMjVicE1QYVFmQllOUDFudU5kSzlQYVBLRFkrRUgyZUlMZTkwPSIsImFsZyI6IlJTMjU2In0.eyJhdF9oYXNoIjoiU3prbUI1ZFFhYl9hUERWQUlndG4yZyIsInN1YiI6IjMyMTZmMDRmLWRiMzAtNDQ0OC1hZjNhLTA0ZTRhNThkZWRkNyIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJpc3MiOiJodHRwczpcL1wvY29nbml0by1pZHAudXMtZWFzdC0xLmFtYXpvbmF3cy5jb21cL3VzLWVhc3QtMV9nU3VUckxZRHQiLCJjb2duaXRvOnVzZXJuYW1lIjoiMzIxNmYwNGYtZGIzMC00NDQ4LWFmM2EtMDRlNGE1OGRlZGQ3Iiwib3JpZ2luX2p0aSI6IjUxMjJjODI5LTIzYTUtNGRjYS1iNTRlLTQwODBhZDhmZjY0YiIsImF1ZCI6IjVvODM1ZXNkZnFrNW4zMzBtM2dpZ240MDBzIiwiZXZlbnRfaWQiOiJkYWEzMDYxZS1kNTI2LTQwOTAtYmU4Yy05YjJiNWQwMDk0NzMiLCJ0b2tlbl91c2UiOiJpZCIsImF1dGhfdGltZSI6MTY1ODI1ODE4MywibmFtZSI6IlRlc3QgcGV0aXRpb25zY2xlcmsxIiwiZXhwIjoxNjU4MjYxNzgzLCJjdXN0b206cm9sZSI6InBldGl0aW9uc2NsZXJrIiwiaWF0IjoxNjU4MjU4MTgzLCJqdGkiOiI0YjkzN2YxOS1kYTZiLTQ4ZTEtYjM2ZS00NjE5NGIxNzgwZDgiLCJlbWFpbCI6InBldGl0aW9uc2NsZXJrMUBleGFtcGxlLmNvbSJ9.dVFe2xw4xzNVokbkbu838pasM-ObU5yw5P5g-UArPtZDI1G1-bLbD5auIugTO9FRKbBoCOr3X28BO1bhO1sBvYE8F3td_j8g8OCTOZqpo1TKe2JV4DpSxQE_ut0RkBxUSO-2mwsdUTbKCZxA5oR15Dv6U65ZZ24YmqbkqUxMG-ZXta2Kd86Bk6ZOD2YMbA6aDBWhGP45qzzS6tmFqCkMKw4wf_3gTZXRviL2JX163lzfgETxhbZsC83Vdw2CVWZ0mFcbCEvmkqmD8OSSNjPquHCW3iNRkfZcxlLlFbz_sWA8-fa-qKzjXeYovet5ooyWZ8OXwxjq5X0DOJ6VbQstMA"
```

Comment:
Should this token be removed, since it's surely expired by now? Replaced with something like `"PLACE TOKEN HERE"` like in the other script?

Reply:
Addressed!